src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -704,6 +704,18 @@ void TwitchMessageBuilder::appendUsername()
         break;
     }
 
+    auto aliases = *getCSettings().aliasNames.readOnly().get();
+    auto loginLower = this->message().loginName.toLower();
```

Comment:
https://github.com/Chatterino/chatterino2/blob/0d947a07f887ea616a543a19f9cfe462405a609c/src/providers/twitch/TwitchMessageBuilder.cpp#L765

Similar to ignore replace I'd like to avoid race conditions if someone would potentially edit the settings while this is ran.